Automattic/Gravatar-SDK-iOS
# Xcode 16.0

Closes #413

### Description

This updates the repo to use Xcode 16.0 for development and CI.  It includes an update to SwiftFormat, which addresses an issue related to Xcode 16.0

### Testing Steps

- CI should be 🟢 
- Smoke test development
   - Build both demo project targets
   - Build the SDK via the Swift Package
- `make test`
- `make generate`
